src/highlighting.c

```diff
@@ -96,7 +96,8 @@ enum	/* Geany common styling */
 	GCS_LINE_HEIGHT,
 	GCS_CALLTIPS,
 	GCS_INDICATOR_ERROR,
-	GCS_MAX
+	GCS_MAX,
+	SCI_SYMBOL_FOLDING_MARGIN_WIDTH
```

Comment:
`GCS_MAX` needs to be last since it tells the length on the enumeration. New enumerator should be named something like `GCS_FOLDING_MARGIN_WIDTH` or whatever is appropriate.